src/NuGet.Core/NuGet.Common/PublicAPI/net45/PublicAPI.Unshipped.txt

```diff
@@ -2,8 +2,11 @@ NuGet.Common.IPackLogMessage.Framework.get -> NuGet.Frameworks.NuGetFramework
 NuGet.Common.IPackLogMessage.Framework.set -> void
 NuGet.Common.IPackLogMessage.LibraryId.get -> string
 NuGet.Common.IPackLogMessage.LibraryId.set -> void
+NuGet.Common.NuGetLogCode.NU1013 = 1013 -> NuGet.Common.NuGetLogCode
+NuGet.Common.NuGetLogCode.NU1504 = 1504 -> NuGet.Common.NuGetLogCode
 NuGet.Common.PackagingLogMessage.Framework.get -> NuGet.Frameworks.NuGetFramework
 NuGet.Common.PackagingLogMessage.Framework.set -> void
 NuGet.Common.PackagingLogMessage.LibraryId.get -> string
 NuGet.Common.PackagingLogMessage.LibraryId.set -> void
+static NuGet.Common.MSBuildStringUtility.IsFalse(string value) -> bool
```

Comment:
I know you're just adding a related method to an existing public class, so I'm just venting about NuGet's public APIs.  I don't feel like we should be a utility library for developers using the NuGet SDK to parse MSBuild properties. If there's value in that, I'd expect MSBuild themselves to provide such helper methods.

Reply:
I agree, I didn't like this either. We don't make this class available through `InternalsVisibleTo` either.  But we do have shared code that's compiled in so I've moved this to `EqualityUtility` instead.  What do you think?